changes/issue-13809-fix-host-filtering

```diff
@@ -0,0 +1 @@
+* Use the correct set of parameters for SQL statement when the `after` and `order_key` query params are passed. Fixes issue 13809.
```

Comment:
Let's add the API endpoint this is referring to.

Reply:
Done!